### PR TITLE
wasp-open-api v2 helm chart

### PR DESCRIPTION
**Pull request by dblane-digicatapult**

### CODE REVIEW

File: charts/wasp-open-api-v2/templates/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+apiVersion: {{ include "common.capabilities.cronjob.apiVersion" . }}
```
Comment (mattdean-digicatapult) [inline charts/wasp-open-api-v2/templates/cronjob.yaml:1]: Is there a particular bitnami chart you've taken the cronjob from? Just looking to make sure we continue following best practices.

Comment (dblane-digicatapult) [reply charts/wasp-open-api-v2/templates/cronjob.yaml:1]: The etcd chart has a cronjob.

Comment (mattdean-digicatapult) [reply charts/wasp-open-api-v2/templates/cronjob.yaml:1]: Is this copied from that then?

Comment (dblane-digicatapult) [reply charts/wasp-open-api-v2/templates/cronjob.yaml:1]: Heavily inspired by.

**Review by mattdean-digicatapult — APPROVED** [review]
I think this lookis good. Happy to use this as a jumping off point

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**